locales/sv_SE/pcsx2_Main.po

```diff
@@ -567,81 +567,81 @@ msgstr ""
 "Tryck ''Vidare'' för att använda det förvalskonfigurerade "
 "InsticksProgrammet, eller ''Avbryt'' för att stänga %s."
 
-#: pcsx2/gui/AppInit.cpp:557
-msgid "PCSX2 Error: Hardware Deficiency"
-msgstr "PCSX2 Fel: HårdvaruBristfällighet"
-
-#: pcsx2/gui/AppInit.cpp:557 pcsx2/gui/AppInit.cpp:569
+#: pcsx2/gui/AppInit.cpp:554 pcsx2/gui/AppInit.cpp:566
 #, c-format
 msgid "Press OK to close %s."
 msgstr "Tryck ''Vidare'' för att stänga %s."
 
-#: pcsx2/gui/AppInit.cpp:570
+#: pcsx2/gui/AppInit.cpp:554
+msgid "PCSX2 Error: Hardware Deficiency"
+msgstr "PCSX2 Fel: HårdvaruBristfällighet"
```

Comment:
What's the deal with compound words? Just write it out: 'Hårdvaran är bristfällig'.

Reply:
Well I like "HårdvaruBristfällighet", but if you incist then I'll change it.